contribute.py

```diff
@@ -40,6 +40,7 @@ def main():
 
     if repository is not None:
         run(['git', 'remote', 'add', 'origin', repository])
+        run(['git', 'branch', '-M', 'main'])
```

Comment:
Thank you for fixing this. I guess we can generate the original branch with `git init -b main` so that it won't require moving the branch. What do you think?

Reply:
I think that is probably the best solution once everyone is up to date with the latest git version, but it may still leave people behind. (Macs can be quite a headache to get git updated to the latest haha). Are there any noticeable downsides to moving the branch? It's your repo so I'm OK with whatever you decide, just a thought.

Reply:
Merged, thank you!